Branch-SDK-TestBed/src/io/branch/branchandroiddemo/MainActivity.java

```diff
@@ -275,6 +275,20 @@ public void onLinkShareResponse(String sharedLink, String sharedChannel, BranchE
                     @Override
                     public void onChannelSelected(String channelName) {
                     }
+
+                    @Override
+                    public String getSharingTitleForChannel(String channel) {
+                        return channel.contains("messenger") ? "title for messenger" :
+                                channel.contains("slack") ? "title for slack" :
```

Comment:
Seems like this hard codings are accidental. Let us not dependent on any hard coded channel